eng/testing/tests.browser.targets

```diff
@@ -1,69 +1,47 @@
 <Project TreatAsLocalProperty="ArchiveTests">
+  <PropertyGroup>
+    <_UseWasmSymbolicator Condition="'$(TestTrimming)' != 'true'">true</_UseWasmSymbolicator>
+  </PropertyGroup>
+
   <Import Project="tests.wasm.targets" />
   <!-- We need to set this in order to get extensibility on xunit category traits and other arguments we pass down to xunit via MSBuild properties -->
   <PropertyGroup>
     <IsBrowserWasmProject Condition="'$(IsBrowserWasmProject)' == ''">true</IsBrowserWasmProject>
-    <WasmGenerateAppBundle Condition="'$(WasmGenerateAppBundle)' == ''">true</WasmGenerateAppBundle>
```

Comment:
I focused mainly on removals. So far that's what I understood:
- `WasmGenerateAppBundle` defaults to true anyway, ok
- `ArchiveTests` - this I don't understand, it's used e.g. by `src/libraries/tests.proj`, won't deleting it change the behavior?
- `BundleTestAppTargets` - looks like it's never read, was only set, ok
- `DebuggerSupport` - is there a sense of letting `DebuggerSupport` to be true for Release?